src/editor/contentState/codeBlockCtrl.js

```diff
@@ -4,7 +4,7 @@ import { createInputInCodeBlock } from '../utils/domManipulate'
 import { escapeInBlockHtml } from '../utils'
 import { codeMirrorConfig, CLASS_OR_ID, BLOCK_TYPE7, DOMPURIFY_CONFIG } from '../config'
 
-const DOMPurify = createDOMPurify(window)
```

Comment:
Why delete `window`?

Reply:
Sorry, my bad. I thought it's unnecessary, but it isn't.